src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, changeQuery] = useState('');
+
+  const isMatching = (prop: string, value: string) => {
+    return prop.toLowerCase().match(value.toLowerCase());
```

Comment:
will be better to create separate variable for storing query in the lower case before the calling filter method